omod/src/main/webapp/pages/results.gsp

```diff
@@ -143,8 +204,9 @@ def prettify = { "$it".replaceAll("null", "").replaceAll("\\s*,\\s", " ") }
 <table id="list-results" cellspacing="0" cellpadding="2" width="100%" class="dataTable">
```

Comment:
@shivtej1505 Are the cellspacing, cell padding and width attributes used with other tables like FindPatient and within Admin management? These have to be kept consistent